src/tree.cc

```diff
@@ -196,7 +196,7 @@ void Tree::GetEditedRange(const Nan::FunctionCallbackInfo<Value> &info) {
 
 void Tree::PrintDotGraph(const Nan::FunctionCallbackInfo<Value> &info) {
   Tree *tree = ObjectWrap::Unwrap<Tree>(info.This());
-  ts_tree_print_dot_graph(tree->tree_, stderr);
+  ts_tree_print_dot_graph(tree->tree_, 2);
```

Comment:
Changed in https://github.com/tree-sitter/tree-sitter/commit/97fd990822deeb3c288f4999a8410fba69f230b6